### DISCUSSION TITLE
LaserGRBL and the demise of Windows

### DISCUSSION THREAD

**Discussion by Bluerdg** [arkypita/LaserGRBL#2299] [General]
If I missed a current discussion please point me to it,  I really like LaserGRBL, but Microsoft is out of touch with reality extorting all of us with the end of support for Windows 10.  With the issues with "Recall"  I'll be skipping 11 until they deal with the privacy issues and migrating my systems to Linux.  Is there any movement toward development of a clean Linux version of the program?  I have spent the last 2 days following all the direction related to Wine and getting nowhere.

**Comment by misterunix** [comment]
Considering the software is written in C#, I doubt a full Linux release will ever exist. Linux, Wine & .net just isn't up to par. That is if a redesign, and he doesn't use the advanced features of C#. It would be a development nightmare for him.
I currently have an issue open #2297 that you might want to follow. The author is working with me to solve a Linux raster load issue.

Bejay
